docs/docs/Develop/install-custom-dependencies.md

```
@@ -23,7 +23,22 @@ To install multiple extras, use commas to separate each dependency group:
 uv pip install "langflow[deploy,local,postgresql]"
 ```
 
-## Install custom dependencies
+## Install custom dependencies in Langflow Desktop {#langflow-desktop}
+
+To add dependencies to **Langflow Desktop**, add an entry for the package to the application's `requirements.txt` file.
+
+    * On macOS, the file is located at `/Users/USER/.langflow/data/requirements.txt`.
```

markdownlint-cli2 (0.17.2)

Code block style
Expected: fenced; Actual: indented

(MD046, code-block-style)